.github/workflows/release-artifacts.yml

```
@@ -53,11 +53,28 @@ jobs:
       with:
         name: packaged-deb
         path: target/debian/*.deb
+        
+  build-rpm:
+    name: 'Build RPM packages'
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        distros: ['fedora:32', 'fedora:33', 'almalinux:8']
```

I suggest that the RPM packages for Fedora are targeting later releases. Fedora 33 is in meantime EOL for some months.

```suggestion
        distros: ['fedora:34', 'fedora:35', 'almalinux:8']
```